pkl-gradle/src/main/java/org/pkl/gradle/spec/ModulesSpec.java

```diff
@@ -23,6 +23,8 @@
 public interface ModulesSpec extends BasePklSpec {
   ListProperty<Object> getSourceModules();
 
+  /** As of Pkl 0.27, this setting is ignored. */
```

Comment:
Would help to explain how users should deal with this change. Presumably this is just inferred automatically now?

Reply:
Actually, I think I'm going to bring this back. Analyzing imports is an extra step that adds time to the best case, that maybe users want to opt-out of.

I think the behavior should be: if not set, compute the import graph.

Reply:
Giving users the option to trade correctness for performance sounds a bit scary. Is it worth it?

Reply:
If users want to specify transitive modules on their own, they can provide a very coarse set, something like this (as you pointed out actually):

```
transitiveModules = fileTree("src") { include("**/*.pkl" }
```

It sounds like a good idea to compute this by default, but provide knobs for users that know what they're doing and want to override it. Does that still sound scary?

Reply:
I just wonder if it's worth keeping a manual knob that adds complexity and can cause correctness issues. I can't really tell, just wanted to bring it up.

Reply:
Our ANTLR parser is quite slow (it takes ~170ms to parse `pkl:base` with jpkl on my machine). We're looking into swapping this out with a hand-rolled parser. Until then, running this as an extra step can be painful for some users, especially if there's no way to opt-out.